Crypto101.org

```diff
@@ -3026,6 +3026,22 @@ Somitra Kumar Sanadhya and Palash Sarkar were able to cause collisions with
 SHA-256 using 24 of 64 rounds (removing the last 40 rounds).
 \cite{eprint-2008-18172}
 
+*** Keccak and SHA-3
+
+Keccak is a family of sponge functions designed by Guido Bertoni, Joan Daemen, Gilles Van Assche and Michaël Peeters, which won NIST's Secure Hash Algorithm Competition in 2012. Keccak has since been standardized in form of the SHA3-224, SHA3-256, SHA3-384 and SHA3-512 hash functions.
+
+Although SHA-3 sounds like it might come from the same family as SHA-2, the two are designed very differently. SHA-3 is very efficient in hardware, but is relatively slow in software in comparison to SHA-2. Security wise, there is nothing to say that SHA-3 is more secure than SHA-2. It is simply a possible replacement in case SHA-2 is broken.
```

Comment:
This makes some claims I'm not sure I'm comfortable with as-is.

Performance numbers should be cited. If security claims are made, they should be put into context. While I generally don't think length-extension attacks on Merkle-Damgard hashes are a flaw in the hash function, it's important to note that a) NIST considered a drastically different design to be intrinsically desirable, and b) not being vulnerable to length-extension attacks (a fortiori: having a direct PRF mode) was considered a conditio sine qua non for SHA-3 contestants.